example/readFromString/readFromString.cpp

```diff
@@ -0,0 +1,35 @@
+#include "json/json.h"
+/*
+parse a string to Value object with CharReaderBuilder class or Reader class
+>g++ readFromString.cpp -ljsoncpp -std=c++11 -o readFromString
+>./readFromString
+colin
+20
+*/
+int main() {
+  std::string strRes = "{\"Age\": 20, \"Name\": \"colin\"}";
+  int nLen = (int)strRes.length();
+  const char* pStart = strRes.c_str();
+
+  JSONCPP_STRING errs;
+  Json::Value root;
+
+#if 0 // old way
+    Json::Reader myreader;
+    myreader.parse(strRes,root);
+#else // new way
+  Json::CharReaderBuilder jsonreader;
```

Comment:
misnamed. It's a builder for a reader, not a reader. Also should follow camelcase.